releasenotes/notes/fix-angular-momentum-73eca0a5afb70679.yaml

```
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    Added the :meth:`.get_overlap_ab_from_qcschema` method which extracts the
```

```suggestion
    Added the :func:`.get_overlap_ab_from_qcschema` function which extracts the
```
This ref did not come out - I imagine it needs to be a function ref but I did not try.